astroplan/core.py

```diff
@@ -2,6 +2,21 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+from astropy.coordinates import (EarthLocation, Latitude, Longitude, SkyCoord,
```

Comment:
Just adding a line note to remind us that this was resolved by a later commit -- the issue was a missing indentation in `astroplan/__init__.py` for a line that was importing from `astroplan.core`.